app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    some_list = []
+    for person_data in people:
+        some_list.append(
+            Person(name=person_data["name"], age=person_data["age"]))
+
+    for person_data in people:
+        person = Person.people[person_data["name"]]
```

Comment:
This line assumes that the person's name is always present in the `Person.people` dictionary. If a name is not found, this will raise a `KeyError`. Consider adding error handling to manage such cases.